### ISSUE TITLE
Startup problem with quasar/src/lang

### ISSUE THREAD

**Issue by Asxer** [memotux/nuxt-quasar#4]
When I started the project by instruction, I got the following error

```
ERROR  Pre-transform error: Failed to resolve import "quasar/src/lang" from ".nuxt/plugins/quasarVueUse.ts". Does the file exist?
```

![image](https://github.com/memotux/nuxt-quasar/assets/6561224/2f9f10e1-a3df-4709-a21a-82c5a79038d2)

Full output: 
```
 > dev
 > nuxt dev

Nuxt 3.11.2

   ➜ Local:    http://localhost:3000/
   ➜ Network:  use --host to expose
 
   ➜ DevTools: press Shift + Alt + D in the browser (v1.2.0)
 
 
  ERROR  Pre-transform error: Failed to resolve import "quasar/src/lang" from ".nuxt/plugins/quasarVueUse.ts". Does the file exist?

 ℹ Vite server warmed up in 12028ms
 ℹ Vite client warmed up in 13325ms
 [nitro] ✔ Nuxt Nitro server built in 5791 ms

  ERROR  (node:50) ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time
 (Use node --trace-warnings ... to show where the warning was created)

```

**Comment by memotux** [comment]
Can you provide installed Quasar version? Thank you.

**Comment by memotux** [comment]
Fix on [v1.0.14](https://github.com/memotux/nuxt-quasar/releases/tag/v1.0.14)
